scripts/static-styles/config.js

```diff
@@ -153,11 +153,11 @@ export default {
     // }),
     { name: 'hr', component: Hr },
     { name: 'image', component: Image },
-    getComponentInfo(Input, {
-      renderPrefix: PropTypes.element,
-      renderSuffix: PropTypes.element,
-      textAlign: [Input.LEFT, Input.RIGHT]
-    }),
+    // getComponentInfo(Input, {
+    //   renderPrefix: PropTypes.element,
+    //   renderSuffix: PropTypes.element,
+    //   textAlign: ['left', 'right']
+    // }),
```

Comment:
Just curios - why is this commented and not deleted?

Reply:
See https://github.com/sumup-oss/circuit-ui/pull/615#discussion_r443348369.